python/sbp/msg.py

```diff
@@ -42,28 +42,7 @@
 
 
 def try_import_jit():
-  '''
-  Try to import the pre-compiled version of the "jit" libraries for parsing SBP data.
-  This should only fail on systems that do not have support for Numba (or LLVM).  For
-  system where it does fail, we fall back to parsing without the pre-compile (or JIT
-  compiled) module.
-  '''
-  parse_jit_name = "parse_jit_py{}".format(str(sys.version_info[0]) + str(sys.version_info[1]))
-  if parse_jit_name in (name for loader, name, ispkg in iter_modules()):
-    # found in sys.path
-    parse_jit = importlib.import_module(parse_jit_name)
-  elif parse_jit_name in (name for loader, name, ispkg in iter_modules(['sbp/jit'])):
-    # found in sbp.jit
-    parse_jit = importlib.import_module('sbp.jit.' + parse_jit_name)
-  else:
-    # not found -> compile
-    try:
-      from sbp.jit import parse
-      parse.compile()
-      parse_jit = importlib.import_module('sbp.jit.' + parse_jit_name)
-    except ImportError:
-      return None
-  return parse_jit
+  return None
```

Comment:
I've kept `try_import_jit` to avoid unnecessarily breaking any code elsewhere which might try to invoke this.  It could emit a UserWarning, and in time become an exception.

It also wouldnt hurt to do something similar to aide deprecation with `sbp.jit`, creating a very small `sbp/jit.py` that emits an error on import, and re-implements any names which can be safely re-implemented without the jit.

Reply:
I like the UserWarning idea and creating a small sbp.jit with a similar purpose sounds reasonable